tests/dbus/test_dbus_client.cpp

```diff
@@ -54,6 +54,11 @@ using otbr::DBus::LinkModeConfig;
 using otbr::DBus::OnMeshPrefix;
 using otbr::DBus::SrpServerInfo;
 using otbr::DBus::ThreadApiDBus;
+
+#if OTBR_ENABLE_TREL
+using otbr::DBus::TrelInfo;
+#endif
```

Comment:
I think the macro guard is unnecessary.

Reply:
Like OTBR_ENABLE_NAT64 and OTBR_ENABLE_DNSSD_DISCOVERY_PROXY below, I would like to keep this convention.